cmd/monaco/main.go

```diff
@@ -61,20 +63,20 @@ func RunImpl(args []string, fileReader util.FileReader) (statusCode int) {
 
 	statusCode = 0
 
-	dryRun, verbose, environments, projectNameToDeploy, path, errorList, flagError := parseInputCommand(args, fileReader)
+	input, err := parseInputCommand(args, fileReader)
```

Comment:
I think you're a rebase away from @patrickpichler 's input handling/CLI changes. 
I assume there will be merge conflicts in this file...? @patrickpichler can maybe help out with that if anything is too entangled